modules/vuetify/index.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path')
+
+module.exports = function nuxtVuetify (options) {
+  // Add assets
+  this.options.head.link.push({
+    rel: 'stylesheet',
+    type: 'text/css',
+    href: '//fonts.googleapis.com/css?family=Material+Icons'
```

Comment:
Yes, I will be adding an option in the upcoming days.

I'm thinking it should be on by default since some components would break without any icons, what do you guys think?